ocs_ci/templates/workloads/cosbench/cosbench_pod.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: cosbench-pod
+  namespace: cosbench-project
+spec:
+  securityContext:
+    fsGroup: 2000
+  serviceAccountName: default
+  containers:
+  - name: cosbench
+    image: quay.io/parikshith/noobaa  #need to push image to qe repo
```

Comment:
There is a docker image available to use: docker pull nexenta/cosbench:latest. Can we use this image?

Reply:
Yes, we can use from docker image but number of pulls is limited. Quay is recommended.